styles/prism.css

```diff
@@ -1,144 +1,166 @@
+pre[class*="language-"]::selection,
+pre[class*="language-"] ::selection,
+code[class*="language-"]::selection,
+code[class*="language-"] ::selection {
+	text-shadow: none;
+	background: rgba(29, 59, 83, 0.99);
 }
 
-.code-line.inserted {
-  @apply su-bg-digital-green su-bg-opacity-20;
+@media print {
+	code[class*="language-"],
+	pre[class*="language-"] {
+		text-shadow: none;
+	}
 }
 
-.code-line.deleted {
-  @apply su-bg-digital-red-light su-bg-opacity-20;
+/* Code blocks */
+pre[class*="language-"] {
+	padding: 1em;
+	margin: 0.5em 0;
+	overflow: auto;
 }
 
-.highlight-line {
-  @apply su--mx-4 su-border-l-4 su-border-digital-red su-bg-black-70 su-bg-opacity-50;
+:not(pre) > code[class*="language-"],
+pre[class*="language-"] {
+	color: white;
+	background: #011627;
 }
 
-.line-number::before {
-  @apply su-mr-4 su--ml-2 su-inline-block su-w-4 su-text-right su-text-black-70;
-  content: attr(line);
+:not(pre) > code[class*="language-"] {
+	padding: 0.1em;
+	border-radius: 0.3em;
+	white-space: normal;
 }
 
-/* Token styles */
-/**
- * MIT License
- * Copyright (c) 2018 Sarah Drasner
- * Sarah Drasner's[@sdras] Night Owl
- * Ported by Sara vieria [@SaraVieira]
- * Added by Souvik Mandal [@SimpleIndian]
- */
 .token.comment,
 .token.prolog,
 .token.cdata {
-  color: rgb(99, 119, 119);
-  font-style: italic;
+	color: rgb(99, 119, 119);
+	font-style: italic;
 }
 
 .token.punctuation {
-  color: rgb(199, 146, 234);
+	color: rgb(199, 146, 234);
 }
 
 .namespace {
-  color: rgb(178, 204, 214);
+	color: rgb(178, 204, 214);
 }
 
 .token.deleted {
-  color: rgba(239, 83, 80, 0.56);
-  font-style: italic;
+	color: rgba(239, 83, 80, 0.56);
+	font-style: italic;
 }
 
 .token.symbol,
 .token.property {
-  color: rgb(128, 203, 196);
+	color: rgb(128, 203, 196);
 }
 
 .token.tag,
 .token.operator,
 .token.keyword {
-  color: rgb(127, 219, 202);
+	color: rgb(127, 219, 202);
 }
 
 .token.boolean {
-  color: rgb(255, 88, 116);
+	color: rgb(255, 88, 116);
 }
 
 .token.number {
-  color: rgb(247, 140, 108);
+	color: rgb(247, 140, 108);
 }
 
 .token.constant,
 .token.function,
 .token.builtin,
 .token.char {
-  color: rgb(130, 170, 255);
+	color: rgb(130, 170, 255);
 }
 
 .token.selector,
 .token.doctype {
-  color: rgb(199, 146, 234);
-  font-style: italic;
+	color: rgb(199, 146, 234);
+	font-style: italic;
 }
 
 .token.attr-name,
 .token.inserted {
-  color: rgb(173, 219, 103);
-  font-style: italic;
+	color: rgb(173, 219, 103);
+	font-style: italic;
 }
 
 .token.string,
 .token.url,
 .token.entity,
 .language-css .token.string,
 .style .token.string {
-  color: rgb(173, 219, 103);
+	color: rgb(173, 219, 103);
 }
 
 .token.class-name,
 .token.atrule,
 .token.attr-value {
-  color: rgb(255, 203, 139);
+	color: rgb(255, 203, 139);
 }
 
 .token.regex,
 .token.important,
 .token.variable {
-  color: rgb(214, 222, 235);
+	color: rgb(214, 222, 235);
 }
 
 .token.important,
 .token.bold {
-  font-weight: bold;
+	font-weight: bold;
```

Comment:
These were added by running eslint - we can change the rules later. I copied the rules from My account